frontend/packages/dev-console/src/components/edit-deployment/deployment-strategy/utils/types.ts

```diff
@@ -8,42 +9,48 @@ export enum DeploymentStrategyType {
   rollingUpdate = 'RollingUpdate',
 }
 
-export const KubernetesDeploymentStrategyOptions = {
-  [DeploymentStrategyType.recreateParams]: 'Recreate',
-  [DeploymentStrategyType.rollingUpdate]: 'Rolling Update',
-};
+export const getKubernetesDeploymentStrategyOptions = () => ({
```

Comment:
@spadgett can you confirm these function wrappers are sufficient to re-evaluate `i18n.t` calls?